lib/OpenQA/WebSockets.pm

```diff
@@ -119,14 +119,38 @@ sub ws_send_job {
     return $result;
 }
 
+sub _check_for_stale_jobs_in_two_minutes {
+    my ($self) = @_;
+
+    Mojo::IOLoop->timer(
+        120 => sub {
+            my $ok = $self->status->workers_checker(
+                sub {
+                    my ($ua, $tx) = @_;
+
+                    if (my $err = $tx->error) {
+                        my $message
+                          = $err->{code}
+                          ? "$err->{code} response: $err->{message}"
+                          : "connection error: $err->{message}";
+                        log_error("Unable to report stale jobs to scheduler ($message)");
+                        if (my $res = $tx->res) { log_error('response was: ' . $res->body); }
+                    }
+
+                    $self->_check_for_stale_jobs_in_two_minutes;
```

Comment:
I suggest we leave *everything* about jobs to the scheduler and only report stale *workers*

Reply:
makes sense